### DISCUSSION TITLE
Firebase/Firestore Integration

### DISCUSSION THREAD

**Discussion by FooJiaYin** [Budibase/budibase#2494] [Ideas]
Budibase is a great service but doesn't support the Firestore database that I'm currently using. I suggest adding Firebase to the data sources since Firebase is one of the most used BaaS services. This will help a lot of people. Thank you for the great job!

**Comment by shogunpurple** [comment]
Thanks for raising - this one has been asked for quite a bit from our community. https://github.com/Budibase/budibase/discussions/891

We will look to address it in the coming months.

**Reply by davidvm2** [reply]
Any chance it's coming soon?

**Reply by mjashanks** [reply]
It won't be this year unless we have a community contribution - always happy to accept those!

**Reply by davidvm2** [reply]
Thanks!
